app/test/std_sdk_test.go

```diff
@@ -220,6 +221,46 @@ func (s *StandardSDKIntegrationTestSuite) TestStandardSDK() {
 			},
 			expectedCode: abci.CodeTypeOK,
 		},
+		{
+			name: "create forbidden param proposal change",
```

Comment:
[non-blocking]
Should we also add tests for the other forbidden params? Or this test is just to ensure the mechanism is working and we should rely on the other unit test below to make sure the list of forbidden params contains the right ones?

Reply:
> Or this test is just to ensure the mechanism is working and we should rely on the other unit test below to make sure the list of forbidden params contains the right ones?

the thinking was to rely on the unit test for that. The integration test is only to ensure that an error is thrown at the expected times.